src/helpers/figures/charts/scorecard_chart.ts

```diff
@@ -177,6 +233,470 @@ export class ScorecardChart extends AbstractChart {
+      ctx.font = structure.baselineArrow.style.font;
+      ctx.fillStyle = structure.baselineArrow.style.color;
+      ctx.fillText(
+        structure.baselineArrow.text,
+        structure.baselineArrow.position.x,
+        structure.baselineArrow.position.y
+      );
+    }
+
+    // Baseline description
+    if (structure.baselineDescr) {
+      ctx.font = structure.baselineDescr.style.font;
+      ctx.fillStyle = structure.baselineDescr.style.color;
+      ctx.fillText(
+        structure.baselineDescr.text,
+        structure.baselineDescr.position.x,
+        structure.baselineDescr.position.y
+      );
+    }
+
+    // Key value
+    if (structure.key) {
+      ctx.font = structure.key.style.font;
+      ctx.fillStyle = structure.key.style.color;
+      ctx.fillText(structure.key.text, structure.key.position.x, structure.key.position.y);
+      if (structure.key.style?.strikethrough) {
+        ctx.lineWidth = structure.key.style.lineWidth ?? 1;
+        ctx.strokeStyle = structure.key.style.color;
+        ctx.beginPath();
+        ctx.moveTo(
+          structure.key.style.strikethrough.from.x,
+          structure.key.style.strikethrough.from.y
+        );
+        ctx.lineTo(structure.key.style.strikethrough.to.x, structure.key.style.strikethrough.to.y);
+        ctx.stroke();
+      }
+      if (structure.key.style?.underline) {
+        ctx.lineWidth = structure.key.style.lineWidth ?? 1;
+        ctx.strokeStyle = structure.key.style.color;
+        ctx.beginPath();
+        ctx.moveTo(structure.key.style.underline.from.x, structure.key.style.underline.from.y);
+        ctx.lineTo(structure.key.style.underline.to.x, structure.key.style.underline.to.y);
+        ctx.stroke();
+      }
+    }
+  }
+
+  computeChart(): ScorecardChartDesign {
+    const structure: any = {
+      canvas: {
+        width: this.props.figure.width,
+        height: this.props.figure.height,
+        backgroundColor: this.backgroundColor,
+      },
+      title: undefined,
+    };
+    const canvas = this.canvas;
+    canvas.width = this.props.figure.width;
+    canvas.height = this.props.figure.height;
+    const ctx = canvas.getContext("2d")!;
+    const style = this.getTextStyles();
+    ctx.font = `${style.titleStyle.fontSize}px ${DEFAULT_FONT}`;
+    const titleMeasure = ctx.measureText(this.title);
+    const titleHeight =
+      titleMeasure.actualBoundingBoxAscent + titleMeasure.actualBoundingBoxDescent;
+    if (this.title) {
+      structure.title = {
+        text: this.title,
+        style: {
+          font: `${style.titleStyle.fontSize}px ${DEFAULT_FONT}`,
+          color: style.titleStyle.color,
+        },
+        position: {
+          x: this.chartPadding,
+          y: this.chartPadding + titleHeight,
+        },
+      };
+    }
+    ctx.font = `${style.baselineValueStyle.fontSize}px ${DEFAULT_FONT}`;
+    let baselineArrow = "";
+    if (this.baselineArrowDirection === "up") {
+      baselineArrow = "\u{1F871}";
```

Comment:
open question: do we care to be consistend with arrow up/down of CFs ? or we don't care and using this unicode is fine